packages/react-ds/src/components/Input/Input.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { FC } from 'react'
+
+import classNames from 'classnames/bind'
+
+import { IInput } from '@venice/core/models/input.model'
+import styles from '@venice/styles/components/Input.module.scss'
+
+const Input: FC<IInput> = ({
+  id,
+  label,
+  name,
+  style,
+  className,
+  error,
+  ...props
+}: IInput) => {
+  const selfId = id || `input__${name}`
+
+  return (
+    <div
+      className={classNames(
+        styles.container,
+        { [styles.invalid]: !!error },
+        className
+      )}
+      style={style}
+    >
+      {label && (
+        <label htmlFor={selfId} className={styles.label}>
+          {label}
+        </label>
+      )}
+      <input className={styles.input} name={name} id={selfId} {...props} />
+      {/* TODO: Replace for alert component */}
+      {error && (
+        <div className={styles.error} data-testid="error">
```

Comment:
I explain it to @elizeulouzeiro and @hick97, but avoid to use `data-testid`, because if project that consume Venice have other element using `data-testid="error"` we'll have problems.

Make tests using `class`.